src/execution_plan/ops/op_node_by_label_scan.c

```diff
@@ -24,13 +24,21 @@ static inline void NodeByLabelScanToString(const OpBase *ctx, sds *buf) {
 	ScanToString(ctx, buf, op->n.alias, op->n.label);
 }
 
+static void _update_label_id(NodeByLabelScan *op) {
+	if(op->n.label_id != GRAPH_UNKNOWN_LABEL) return;
+
+	GraphContext *gc = QueryCtx_GetGraphCtx();
+	Schema *s = GraphContext_GetSchema(gc, op->n.label, SCHEMA_NODE);
+	if(s != NULL) op->n.label_id = Schema_GetID(s);
+}
+
```

Comment:
Please include a short comment describing the purpose and the reason for this function